content/blog/2021/10/05/OptimizeYourPython.adoc

```diff
@@ -0,0 +1,438 @@
+= Optimize your Python
+Christopher-Chianelli
+2021-10-05
+:page-interpolate: true
+:jbake-type: post
+:jbake-tags: python, integration
+
+OptaPlanner can now be used in Python using https://pypi.org/project/optapy[OptaPy]!
+Let's create the school timetabling example in pure Python. Feel free to follow along in the https://mybinder.org/v2/gh/optapy/optapy/main?filepath=index.ipynb[OptaPy Jupyter notebook].
```

Comment:
Ok, the first paragraph has a few goals that it needs to attain:

- Convince python people to take a look at this. So ROI. The words "use OptaPlanner in Python" isn't convincing for python users (they don't know optaplanner).
- Assure them that they can use plain Python. "Pure python" is good already, but I think "plain Python" and mentioning pype might be better.
- The binder message is very important, but can be in the 2th paragraph. It's no longer "motivation" (1th para), it's a "call to action" (2th para).
- Assure Java users reading this that we're not migrating to Python. Prove that by mentioning we support Kotlin too. Or the perf difference?
- Explain OptaPy vs OptaPlanner. Explain its essentially a wrapper.

Let me think of a proposal text and suggest it here.